public-docs/concepts-inventory.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Inventory
```

Comment:
Does this need an `id` too?

Reply:
Works without it (uses file name) but we do usually include it by convention. I added it.